### PR TITLE
[frameworks][examples] Add RedwoodJS

### PR DESCRIPTION
**Pull request by lachlanjc**
Adding Redwood to the frameworks list, static builders file, & an example app for it.

Waiting on the creation of a Redwood builder to merge.

### CODE REVIEW

File: examples/redwoodjs/redwood.toml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+[web]
+  port = 8910
+  apiProxyPath = "/.netlify/functions"
+[api]
+  port = 8911
+[browser]
+  open = true
```
Comment (styfle) [inline examples/redwoodjs/redwood.toml]:
```suggestion
  open = false
```

File: examples/redwoodjs/redwood.toml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+[web]
+  port = 8910
+  apiProxyPath = "/.netlify/functions"
```
Comment (styfle) [inline examples/redwoodjs/redwood.toml:3]: Do you know what `apiProxyPath` is?

The docs just say "TODO" 🤔 

image

File: examples/redwoodjs/web/src/Routes.js [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+//
+// 'src/pages/HomePage/HomePage.js'         -> HomePage
+// 'src/pages/Admin/BooksPage/BooksPage.js' -> AdminBooksPage
+
+import { Router, Route } from '@redwoodjs/router'
+
+const Routes = () => {
+  return (
+    <Router>
+      <Route notfound page={NotFoundPage} />
```
Comment (styfle) [inline examples/redwoodjs/web/src/Routes.js:15]: I think we need a Home Page here

File: packages/frameworks/frameworks.json [hunk 4]
```diff
@@ -55,6 +55,34 @@
...
+    "demo": "https://redwoodjs.now-examples.now.sh",
+    "logo": "https://raw.githubusercontent.com/vercel/vercel/master/packages/frameworks/logos/redwood.svg",
+    "tagline": "Bringing full-stack to the Jamstack.",
+    "description": "A RedwoodJS app and a Serverless Function API.",
+    "website": "https://redwoodjs.com",
+    "detectors": {
+      "every": [
+        {
+          "path": "package.json",
+          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@redwoodjs\/core\":\\s*\".+?\"[^}]*}"
```
Comment (styfle) [inline packages/frameworks/frameworks.json]:
```suggestion
          "matchContent": "\"(dev)?(d|D)ependencies\":\\s*{[^}]*\"@redwoodjs\\/core\":\\s*\".+?\"[^}]*}"
```

File: examples/redwoodjs/redwood.toml (hunk 2 above)
Comment (lachlanjc) [reply examples/redwoodjs/redwood.toml:3]: No idea…

File: examples/redwoodjs/web/src/Routes.js (hunk 3 above)
Comment (lachlanjc) [reply examples/redwoodjs/web/src/Routes.js:15]: Redwood will display a page similar to the way Rails & Blitz.js do to say it's working. We could add one ourselves to this app but it'll work without as well

File: examples/redwoodjs/redwood.toml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+[web]
+  port = 8910
```
Comment (styfle) [inline examples/redwoodjs/redwood.toml:2]: Is it possible to change the port here to use the `$PORT` environment variable? We need a way for `vercel dev` to assign the port dynamically.

Comment (lachlanjc) [reply examples/redwoodjs/redwood.toml:2]: I’m not sure—there’s a decent chance we’ll need to just talk to the Redwood team directly, they haven’t optimized for anything but Netlify yet

**Comment by styfle** [comment]
Closing in favor of #4915

**Pull request closed without merging**